test/integration_test.js

```diff
@@ -648,6 +648,80 @@ describe("Integration tests", function() {
             });
           });
         });
+
+        describe("Resolving conflicts doesn't interfere with sync", () => {
+          const conflictingId = uuid4();
+          const testData = {
+            localSynced: [
+              {id: conflictingId, title: "conflicting task", done: false}
+            ],
+            localUnsynced: [],
+            server: [],
+          };
+          beforeEach(() => testSync(testData));
+          it("should sync over resolved records", () => {
+            return tasks.update({id: conflictingId, title: "locally changed title"},
+                         {patch: true})
+              .then(newRecord => {
+                expect(newRecord.data.last_modified).to.exist;
+                return tasks.api.bucket("default").collection("tasks").updateRecord(
+                  {id: conflictingId, title: "remotely changed title"}, {patch: true});
+              })
+              .then(() => tasks.sync())
+              .then(syncResult => {
+                expect(syncResult.ok).eql(false);
+                expect(syncResult.conflicts).to.have.length.of(1);
+                // always pick our version, which has an older last_modified
+                return tasks.resolve(syncResult.conflicts[0], syncResult.conflicts[0].local);
+              })
+              .then(() => tasks.sync())
+              .then(syncResult => {
+                expect(syncResult.ok).eql(true);
+                expect(syncResult.conflicts).to.have.length.of(0);
+                expect(syncResult.updated).to.have.length.of(0);
+              })
+              .then(() => tasks.get(conflictingId))
+              .then(record => {
+                expect(record.data.title).eql("locally changed title");
+                expect(record.data._status).eql("synced");
+              });
+          });
+
+          it("should not skip other conflicts", () => {
+            const conflictingId2 = uuid4();
+            const rawCollection = tasks.api.bucket("default").collection("tasks");
+            return tasks.create({id: conflictingId2, title: "second title"},
+                                {useRecordId: true})
+              .then(() => tasks.sync())
+              .then(() =>
+                rawCollection.updateRecord(
+                  {id: conflictingId, title: "remotely changed title"}, {patch: true})
+                   )
+              .then(() =>
+                rawCollection.updateRecord(
+                  {id: conflictingId2, title: "remotely changed title2"}, {patch: true}))
```

Comment:
Yeah, sorry -- I've been working on a team with someone who was very strict about not exceeding 80 characters per line :) Fixed in 1a20231cd5117fda00a7c9b086499404127e9b9a.